### DISCUSSION TITLE
Changing fan mode changes the Power plan to Windows default (balanced)

### DISCUSSION THREAD

**Discussion by Zekkis** [seerge/g-helper#3321] [Q&A]
so ive finally got a code working to cycle through all my power plan with ghelper, but then a slight annoyance. When changing the fan mode with ghelper it automatically changes the power plan to the windows balanced power plan. Is there a way to disable that, more specifically disable changing fan mode also changes power plan to default Windows balanced

**Comment by seerge** [comment]
@Zekkis hello, Balanced plan is being set absolutely intentionally cause it allows to change **Power Modes** in windows along with each performance mode (Silent / Balanced / Turbo).

If you really want to use custom power plan instead - [you can do that as well](https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode)

**Reply by Zekkis** [reply]
thank you for the info. I guess because my windows power plans only have 1 default that is balanced, and my silent and turbo power plan was added by myself, ghelper doesnt read it from the get go, because when i change fan mode to turbo its still on balanced power plan..

Edit: now im more sure about it because only Balanced GUID is the same

**Reply by seerge** [reply]
In modern windows power plans are made almost obsolete and substituted with **Power Modes** (that you select in Settings in windows). Balanced power plan doesn't lock any power modes (unlike some custom plans).
